### DISCUSSION TITLE
Unclear product and documentation

### DISCUSSION THREAD

**Discussion by lcucurachi** [amplication/amplication#7655] [General] [answered]
Hi everyone, sorry to be a bummer but I'm really unclear what the products offers in terms of writing the server code for the micro services. Is this a Node.JS only code builder? Does it support other languages? I find the website unclear. There are a lot of plugins but I cannot see where it talks about the base code in which the micro services are written.

**Comment by overbit** [comment, marked as answer]
Hi @LoKKeR96, atm we are only supporting the generation of Node.JS services as mentioned in our docs https://docs.amplication.com. 

> [Amplication](https://docs.amplication.com/about/) is an open-source backend development tool. It accelerates the development of Node.js applications by auto-generating production-ready backends.
